kalite/control_panel/tests/features/environment.py

```diff
@@ -0,0 +1,63 @@
+"""
+environment.py defines setup and teardown behaviors for behave tests.
+The behavior in this file is appropriate for integration tests, and
+could be used to bootstrap other integration tests in our project.
+It sets up a test server and test database by using the LiveServerTestCase
+machinery.
+"""
+from behave import *
+from httplib import CannotSendRequest
+from selenium import webdriver
+from urlparse import urljoin
+
+from django.test.testcases import LiveServerTestCase
+
+from kalite.testing.behave_helpers import login_as_admin, logout
+
+def before_all(context):
+
+    # So we get a free test server and test database, with appropriate
+    # setup and teardown methods
+    hijacked_test_case = context.hijacked_test_case = HijackTestCase()
+    hijacked_test_case.setUpClass()
+     
+    def browser_url(url):
+        return urljoin(hijacked_test_case.live_server_url, url)
+    
+    context.browser_url = browser_url
+
+
+def after_all(context):
```

Comment:
Seems like some of these functions could be inherited from a base environment - or were you trying to avoid that?